tests/rakshctl/app/create_app_test.go

```diff
@@ -65,26 +65,46 @@ func TestAppCreateUnsupportedWorkload(t *testing.T) {
 		t.Fatalf("Failed to write to %s file", workload)
 	}
 
-	symmKeyFile := dir + "/symm_key"
+	symmKeyFile := dir + "/symmKey"
 
-	f, err := os.Create(symmKeyFile)
+	kf, err := os.Create(symmKeyFile)
 	if err != nil {
 		t.Fatalf("Error to create symmKeyFile file: %+v", err)
 	}
 
 	buf, err := randutil.GetBytes(32)
 	if err != nil {
 		t.Fatalf("Unable to get random bytes for key: %+v", err)
-		f.Close()
+		kf.Close()
 	}
 
-	_, err = f.Write(buf)
+	_, err = kf.Write(buf)
 	if err != nil {
-		t.Fatalf("Error in wirting key to symmKeyFile file: %+v", err)
-		f.Close()
+		t.Fatalf("Error in writing key to symmKeyFile file: %+v", err)
+		kf.Close()
 	}
 
-	var cmdArgs = []string{"app", "create", "-f", workload, "-i", "securecontainerimage-example", "--symmKeyFile", symmKeyFile}
+	nonceFile := dir + "/nonce"
+
+	nf, err := os.Create(nonceFile)
+	if err != nil {
+		t.Fatalf("Error to create nonce file: %+v", err)
+	}
+
+	buf, err = randutil.GetBytes(12)
+	if err != nil {
+		t.Fatalf("Unable to get random bytes for nonce: %+v", err)
+		nf.Close()
+	}
+
+	_, err = nf.Write(buf)
+	if err != nil {
+		t.Fatalf("Error in writing to nonceFile file: %+v", err)
+		nf.Close()
+	}
+
+	var cmdArgs = []string{"app", "create", "-f", workload, "-i", "securecontainerimage-example", "--symmKeyFile", symmKeyFile,
```

Comment:
can we add testcases for `--insecure` flag?

Reply:
yes.. will do